packages/odyssey-react/jest.setup.js

```diff
@@ -10,4 +10,20 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import "@testing-library/jest-dom";
+import 'regenerator-runtime/runtime'
+import '@testing-library/jest-dom';
+import 'jest-axe/extend-expect';
+import { axe } from 'jest-axe';
+
+global.a11yCheck = (renderFn) => {
+  describe('accessibility', () => {
+    it('meets WCAG 2.1 AA criteria', async () => {
+      const { container } = renderFn()
+      const results = await axe(container, {
+        runOnly: ['wcag2a', 'wcag2aa', 'wcag21a', 'wcag21aa']
```

Comment:
[see available options here.](https://github.com/dequelabs/axe-core/blob/master/doc/API.md#options-parameter) I just went with something simple for now

Reply:
Wondering if it worth adding `section508` and keeping these in sync with what we have in okta-ui ? https://github.com/okta/okta-ui/blob/master/packages/core/components/src/utils/test-utils.ts#L27

Reply:
https://github.com/okta/okta-ui-react/blob/master/packages/enduser/test/testcafe/framework/testConstants.ts#L1
here is what okta-ui-react uses

Reply:
WCAG 2.1 includes all criteria from WCAG 2.0, so we should only need `'wcag21a', 'wcag21aa'` (and possibly `section508`).

From WC3:

>Content that conforms to WCAG 2.1 also conforms to WCAG 2.0. And content that conforms to WCAG 2.2 will also conform to 2.1 and 2.0. (This is often called “backwards compatible”.) A website that meets WCAG 2.1 or 2.2 should meet the requirements of policies that reference WCAG 2.0. To put it another way: If you want to meet both WCAG 2.0 and WCAG 2.1, you can use the 2.1 resources and you don’t need to bother looking at 2.0.